src/client/layouts/MinimalLayout.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import MinimalHeader from '@/client/components/MinimalHeader';
+import { from, headline, remSpace } from '@guardian/source/foundations';
+import useClientState from '@/client/lib/hooks/useClientState';
+import {
+	ErrorSummary,
+	SuccessSummary,
+} from '@guardian/source-development-kitchen/react-components';
+import locations from '@/shared/lib/locations';
+import { Theme } from '@/client/styles/Theme';
+import {
+	mainSectionStyles,
+	successMessageStyles,
+} from '@/client/styles/Shared';
+import { DecorativeImageId } from '@/client/assets/decorative';
+import { MinimalLayoutImage } from '@/client/components/MinimalLayoutImage';
+import { LAYOUT_WIDTH_NARROW, LAYOUT_WIDTH_WIDE } from '@/client/models/Style';
+import { MainBodyText } from '@/client/components/MainBodyText';
+import { errorMessageStyles } from '@/client/styles/Shared';
+
+export interface MinimalLayoutProps {
+	children: React.ReactNode;
+	wide?: boolean;
+	pageHeader: string;
+	leadText?: string | React.ReactNode;
```

Comment:
```suggestion
	leadText?: React.ReactNode;
```

ReactNodes can already be of type string amongst others!

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f855c408dac3c7b3bf0ed9d78242ce073c7aaf1/types/react/index.d.ts#L478

Reply:
TIL!